fv3core/stencils/map_single.py

```diff
@@ -297,114 +312,113 @@ def lagrangian_contributions_transliterated(
+                            * (1.0 + pl)
+                            - q4_4[ii, 0, k1] * (r3 * (1.0 + pl * (1.0 + pl)))
+                        )
 
-    #                     for mm in np.arange(k1 + 1, km):  # find the bottom edge
-    #                         if (
-    #                             pe2[ii, 0, k2 + 1] > pe1[ii, 0, mm + 1]
-    #                         ):  # Not there yet; add the whole layer
-    #                             qsum = qsum + dp1[ii, 0, mm] * q4_1[ii, 0, mm]
-    #                         else:
-    #                             dp = pe2[ii, 0, k2 + 1] - pe1[ii, 0, mm]
-    #                             esl = dp / dp1[ii, 0, mm]
-    #                             qsum = qsum + dp * (
-    #                                 q4_2[ii, 0, mm]
-    #                                 + 0.5
-    #                                 * esl
-    #                                 * (
-    #                                     q4_3[ii, 0, mm]
-    #                                     - q4_2[ii, 0, mm]
-    #                                     + q4_4[ii, 0, mm] * (1.0 - r23 * esl)
-    #                                 )
-    #                             )
-    #                             k0 = mm
-    #                             flag = 1
-    #                             elems[ii - i1, k2] = 0
-    #                             break
-    #                     # Add everything up and divide by the pressure difference
-    #                     q2[ii, j_2d, k2] = qsum
-    #                         / (pe2[ii, 0, k2 + 1] - pe2[ii, 0, k2])
-    #                     break
+                        for mm in np.arange(k1 + 1, km):  # find the bottom edge
+                            # Not there yet; add the whole layer
+                            if pe2[ii, 0, k2 + 1] > pe1[ii, 0, mm + 1]:
+                                qsum = qsum + dp1[ii, 0, mm] * q4_1[ii, 0, mm]
+                            else:
+                                dp = pe2[ii, 0, k2 + 1] - pe1[ii, 0, mm]
+                                esl = dp / dp1[ii, 0, mm]
+                                qsum = qsum + dp * (
+                                    q4_2[ii, 0, mm]
+                                    + 0.5
+                                    * esl
+                                    * (
+                                        q4_3[ii, 0, mm]
+                                        - q4_2[ii, 0, mm]
+                                        + q4_4[ii, 0, mm] * (1.0 - r23 * esl)
+                                    )
+                                )
+                                k0 = mm
+                                flag = 1
+                                elems[ii - i1, k2] = 0
+                                break
+                        # Add everything up and divide by the pressure difference
+                        q2[ii, j_2d, k2] = qsum / (pe2[ii, 0, k2 + 1] - pe2[ii, 0, k2])
+                        break
+    """
```

Comment:
Is it worth it keeping this commented out code? I feel like it could be time to say goodbye and cut this out.

Reply:
Could not agree more!